src/libcore/intrinsics.rs

```diff
@@ -707,11 +707,26 @@ extern "rust-intrinsic" {
                          they should be used through stabilized interfaces \
                          in the rest of the standard library",
                issue = "0")]
-    #[rustc_deprecated(reason = "no longer used by rustc, will be removed - use MaybeUninit \
-                                 instead",
+    #[rustc_deprecated(reason = "superseded by MaybeUninit, removal planned",
```

Comment:
Why did you change the message? The new message is fine, but this will make reverting the commit more annoying.

Reply:
To be accurate. The comments and actual code should match, otherwise it would be confusing for someone who stumbled upon this.

Reply:
Fair.